src/printer/index.js

```diff
@@ -779,6 +779,10 @@ function genericPrint(path, options, print) {
       ]);
     }
 
+    case "Set": {
+      return printListLike("{", path.map(print, "elts"), trailingComma, "}");
```

Comment:
This might be a good thing to have in Prettier core — I could see it being used in lots of places.